presto-parser/src/main/java/com/facebook/presto/sql/tree/Select.java

```diff
@@ -64,7 +64,7 @@ public boolean isDistinct()
     @Override
     public List<Node> getChildren()
     {
-        return null;
+        return ImmutableList.copyOf(selectItems);
```

Comment:
Thanks for explanation. I would feel uncomfortable simply returning `selectItems` (what if someone changes field type...) unless the method return type is narrowed to `ImmutableList`. I hope that's OK.